### PR TITLE
Added Profile Page Testcases

### PR DESCRIPTION
**Pull request by venuarelli**
Created Pull request for Profile TestCases.

### CODE REVIEW

File: src/main/java/com/hu/webdriver/pages/BasePage.java [hunk 1]
```diff
@@ -39,8 +44,14 @@ public BasePage(WebDriver driver){
 	public void clearAndType(WebElement element,String text){
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/BasePage.java:44]: Provide documentation all for methods. Whenever its applicable

File: src/main/java/com/hu/webdriver/pages/NewsFeedPage.java [hunk 2]
```diff
@@ -7,21 +7,78 @@
...
+	/**
+	 * WebElement for Logout
+	 */
+	@FindBy(id = "sitebar-logout-button")
+	WebElement logout;
+	/**
+	 * WebElemenet for  
+	 */
+	@FindBy(id = "sitebar-messages-button")
+	WebElement messages;
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/NewsFeedPage.java:48]: WebElement for "Messages"

File: src/main/java/com/hu/webdriver/pages/NewsFeedPage.java [hunk 3]
```diff
@@ -7,21 +7,78 @@
...
+	/**
+	 * Method to get NewsFeed Text.
+	 * @return newsfeed text
+	 */
 	public String getNewsFeedText(){
 		WebDriverWait wait = new WebDriverWait(driver, 30);
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/NewsFeedPage.java:54]: Use findVisibleElement Method

File: src/main/java/com/hu/webdriver/pages/NewsFeedPage.java [hunk 4]
```diff
@@ -7,21 +7,78 @@
...
+		findVisibleElement(By.xpath("//a[@href='/profile/edit']"), 30);
+		updateProfile.click();
+	}
+	/**
+	 * Method to Click On User Menu Drop Down. 
+	 */
+	public void clickOnUserMenuDropDown(){
+		userMenuDropdown.click();
+	}
+	public void clickOnMessages(){
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/NewsFeedPage.java:71]: Comment Provided

File: src/main/java/com/hu/webdriver/pages/BasePage.java [hunk 5]
```diff
@@ -39,8 +44,14 @@ public BasePage(WebDriver driver){
...
 		element.clear();
 		element.sendKeys(text);
-		
 	}
-
+	
+	public void implicityWait(int seconds){
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/BasePage.java:49]: Comment Provided

File: src/main/java/com/hu/webdriver/pages/BasePage.java [hunk 6]
```diff
@@ -39,8 +44,14 @@ public BasePage(WebDriver driver){
...
 		element.clear();
 		element.sendKeys(text);
-		
 	}
-
+	
+	public void implicityWait(int seconds){ 
+		driver.manage().timeouts().implicitlyWait(seconds, TimeUnit.SECONDS);
+	}
+	public void findVisibleElement(By by,int seconds){
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/BasePage.java:52]: Comment Provided

File: src/main/java/com/hu/webdriver/pages/home/HUHomePage.java [hunk 7]
```diff
@@ -11,17 +12,25 @@
 public class HUHomePage extends BasePage{
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/home/HUHomePage.java:12]: Add author doc

File: src/main/java/com/hu/webdriver/pages/login/HULoginPage.java [hunk 8]
```diff
@@ -14,22 +14,39 @@
...
 	}
+	/**
+	 * Method to Set Password
+	 * @param password
+	 */
 	public void setPassword(String password){
 		clearAndType(this.password, password);
 	}
+	/**
+	 * Method to Click On Submit Button and Navigate to NewsFedd Page
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/login/HULoginPage.java:47]: "NewsFedd"------>"NewsFeed"

File: src/main/java/com/hu/webdriver/pages/login/HULoginPage.java [hunk 9]
```diff
@@ -14,22 +14,39 @@
...
+	 * @return mewsfeedPage
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/login/HULoginPage.java:48]: "mewsfeedPage"------->"newsfeedPage"

File: src/main/java/com/hu/webdriver/pages/messages/MessagesPage.java [hunk 10]
```diff
@@ -0,0 +1,13 @@
+package com.hu.webdriver.pages.messages;
+
+import org.openqa.selenium.WebDriver;
+
+import com.hu.webdriver.pages.BasePage;
+
+public class MessagesPage extends BasePage{
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/messages/MessagesPage.java:7]: Add author doc

File: src/main/java/com/hu/webdriver/pages/messages/MessagesPage.java [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+
+	public MessagesPage(WebDriver driver) {
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/messages/MessagesPage.java:9]: Provide method doc

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 12]
```diff
@@ -0,0 +1,356 @@
...
+	 */
+	@FindBy(xpath = "(//span[@class='icon-down-open-1'])[1]")
+	WebElement dropDownForProfile;
+	/**
+	 * WebElement for Profile
+	 */
+	@FindBy(id = "sitebar-profile-button")
+	WebElement profile;
+	/**
+	 * WebElemnt for following
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:55]: "WebElemnt"---->"WebElement"

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 13]
```diff
@@ -0,0 +1,356 @@
...
+	 */
+	@FindBy(name = "limitedTextArea")
+	WebElement fillYourBio;
+	/**
+	 * WebElement For Save Changes
+	 */
+	@FindBy(xpath = "//input[@value='Save changes']")
+	WebElement saveChanges;
+	/**
+	 * WebElemnt for DropDown For Profile
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:45]: "WebElemnt"---->"WebElement"

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 14]
```diff
@@ -0,0 +1,356 @@
...
+	WebElement yearDropDown;
+	/**
+	 * WebElement for Country DropDown
+	 */
+	@FindBy(xpath = "(//select[contains(class,day)])[4]")
+	WebElement countryDropDown;
+	/**
+	 * WebElement for ProfilePageText
+	 */
+	@FindBy(xpath = "//span[text()='Hi! I’m aarellivenu']")
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:127]: Don't hardcode profile names, Use it from properties file.

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 15]
```diff
@@ -0,0 +1,356 @@
...
+	 * Method to click On UpdateProfile
+	 */
+	public void clickOnUpdateProfile() {
+		findVisibleElement(By.xpath("//a[@href='/profile/edit']"), 30);
+		updateProfile.click();
+	}
+
+	/**
+	 * Method to Fill Your Bio
+	 */
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:161]: @param doc missing

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 16]
```diff
@@ -0,0 +1,356 @@
...
+	 * Method to Click On SavChanges Button
+	 */
+	public void clickOnSaveChangesButton() {
+		saveChanges.click();
+	}
+
+	/**
+	 * Method to Edit Your Bio
+	 */
+	public void editYourBio(String editBIO) {
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:177]: Same as above

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 17]
```diff
@@ -0,0 +1,356 @@
...
+	 * @return text
+	 */
+	public String getFollowText() {
+		String text = followButton.getText();
+		return text;
+	}
+
+	/**
+	 * Method to Enter Symptom
+	 */
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:229]: same as above

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 18]
```diff
@@ -0,0 +1,356 @@
...
+	public void enterSymptom(String symptom) {
+		clearAndType(this.symptom, symptom);
+	}
+
+	/**
+	 * Method to Enter Condition
+	 */
+	public void enterCondition(String condition) {
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:237]: same as above

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 19]
```diff
@@ -0,0 +1,356 @@
...
+		clearAndType(this.condition, condition);
+	}
+
+	/**
+	 * Method to Enter Treatment
+	 */
+	public void enterTreatement(String treatement) {
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:244]: same as above

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 20]
```diff
@@ -0,0 +1,356 @@
...
+	 */
+	public String getMethotrexateText() {
+		String text = methotrexateText.getText();
+		return text;
+	}
+
+	/**
+	 * Method to Select Date Drop Down
+	 */
+	public void selectDateDropDown(String date) {
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:281]: same as above

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 21]
```diff
@@ -0,0 +1,356 @@
...
+		findVisibleElement(By.xpath("(//select[contains(class,day)])[1]"), 30);
+		Select select = new Select(dateDropDown);
+		select.selectByValue(date);
+	}
+
+	/**
+	 * Method to Select Month Drop Down
+	 */
+	public void selectMonthDropDown(String month) {
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:290]: same as above

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 22]
```diff
@@ -0,0 +1,356 @@
...
+		findVisibleElement(By.xpath("(//select[contains(class,month)])[2]"), 30);
+		Select select = new Select(monthDropDown);
+		select.selectByValue(month);
+	}
+
+	/**
+	 * Method to Slect Year Drop down
+	 */
+	public void selectYearDropDown(String year) {
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:299]: same as above

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 23]
```diff
@@ -0,0 +1,356 @@
...
+		findVisibleElement(By.xpath("(//select[contains(class,year)])[3]"), 30);
+		Select select = new Select(yearDropDown);
+		select.selectByVisibleText(year);
+	}
+
+	/**
+	 * Method to Select Country Drop Down
+	 */
+	public void selectCountryDropDown() {
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:308]: Don't hard code values, Use it from properties file by taking as param..

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 24]
```diff
@@ -0,0 +1,356 @@
...
+		select.selectByVisibleText("India");
+	}
+
+	/**
+	 * Method to getProfilePageText
+	 * 
+	 * @return
+	 */
+	public String getProfilePageText() {
+		findVisibleElement(By.xpath("//span[text()='Hi! I’m aarellivenu']"), 30);
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:319]: Commented provided for WebElement

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 25]
```diff
@@ -0,0 +1,356 @@
...
+		String text = profilePageText.getText();
+		return text;
+	}
+
+	/**
+	 * Methdo to Set Gender
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:325]: Methdo --->Method

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 26]
```diff
@@ -0,0 +1,356 @@
...
+	/**
+	 * Method to Select Country Drop Down
+	 */
+	public void selectCountryDropDown() {
+		Select select = new Select(countryDropDown);
+		select.selectByVisibleText("India");
+	}
+
+	/**
+	 * Method to getProfilePageText
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:314]: Provide clear description

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java [hunk 27]
```diff
@@ -0,0 +1,356 @@
...
+	 */
+	public void setGender() {
+		findVisibleElement(By.xpath("//span[text()='Male']"), 30);
+		male.click();
+	}
+
+	/**
+	 * Method to Set Postal Code
+	 */
+	public void setPostalCode() {
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:335]: Don't hard code values, Use it from properties by passing as param

File: src/main/java/com/hu/webdriver/pages/profile/UpDateProfilePage.java [hunk 28]
```diff
@@ -0,0 +1,105 @@
...
+	/**
+	 * ConStructor to Initialize UpDateProfilePage.
+	 * @param driver
+	 */
+	public UpDateProfilePage(WebDriver driver) {
+		super(driver);
+		PageFactory.initElements(driver, this);
+	}
+	/**
+	 * Method to Get BIOText
```
Comment (VenkiDeveloper) [inline src/main/java/com/hu/webdriver/pages/profile/UpDateProfilePage.java:56]: @return doc is missing..

File: src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java [hunk 29]
```diff
@@ -0,0 +1,44 @@
...
+import org.testng.annotations.AfterClass;
+
+import com.hu.webdriver.pages.NewsFeedPage;
+import com.hu.webdriver.pages.home.HUHomePage;
+import com.hu.webdriver.pages.login.HULoginPage;
+import com.hu.webdriver.pages.messages.MessagesPage;
+import com.hu.webdriver.pages.profile.ProfilePage;
+import com.hu.webdriver.tests.messages.MessagesTest;
+
+public class BaseLoginWebDriverTest extends BaseTest{
```
Comment (VenkiDeveloper) [inline src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java:12]: Provide documentation for class

File: src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java [hunk 30]
```diff
@@ -0,0 +1,44 @@
...
+	
+	protected NewsFeedPage newsFeedPage;
+	/**
+	 * Method to login HU and navigate to News feed page.
+	 * @return
+	 */
+	public NewsFeedPage loginHUAndNavigateToNewsFeedPage(){
```
Comment (VenkiDeveloper) [inline src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java:19]: Return description missing

File: src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java [hunk 31]
```diff
@@ -0,0 +1,44 @@
...
+		HUHomePage homePage = new HUHomePage(webDriver);
+		HULoginPage loginPage = homePage.clickOnLoginButton();
+		loginPage.loginHU();
+		return new NewsFeedPage(webDriver);
+	}
+	
+	public ProfilePage loginHUAndNavigateToProfilePage(){
```
Comment (VenkiDeveloper) [inline src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java:26]: Updated method docs

File: src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java [hunk 32]
```diff
@@ -0,0 +1,44 @@
...
+		NewsFeedPage newsFeedPage = loginHUAndNavigateToNewsFeedPage();
+		newsFeedPage.clickOnUpdateProfile();
+		return new ProfilePage(webDriver);
+	}
+	
+	public MessagesPage loginHuAndNavigateToMessagesPage(){
```
Comment (VenkiDeveloper) [inline src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java:32]: Same as above

File: src/test/java/com/hu/webdriver/tests/BaseTest.java [hunk 33]
```diff
@@ -86,15 +83,4 @@ public void beforeTest(){
 	public void logOutFromHU(){
```
Comment (VenkiDeveloper) [inline src/test/java/com/hu/webdriver/tests/BaseTest.java:83]: Method doc

File: src/test/java/com/hu/webdriver/tests/profile/ProfilePageTest.java [hunk 34]
```diff
@@ -0,0 +1,162 @@
...
+import com.hu.webdriver.pages.profile.UpDateProfilePage;
+import com.hu.webdriver.tests.BaseLoginWebDriverTest;
+
+/** 
+ * This Class Contains ProfilePage features
+ * @author Venu
+ *
+ */
+public class ProfilePageTest extends BaseLoginWebDriverTest{
+	ProfilePage profilePage;
```
Comment (VenkiDeveloper) [inline src/test/java/com/hu/webdriver/tests/profile/ProfilePageTest.java:17]: Instance Variable for Profile page.

File: src/test/java/com/hu/webdriver/tests/profile/ProfilePageTest.java [hunk 35]
```diff
@@ -0,0 +1,162 @@
...
+	@BeforeClass
```
Comment (VenkiDeveloper) [inline src/test/java/com/hu/webdriver/tests/profile/ProfilePageTest.java:18]: Before class to login and Navigate to Profile page.

Changed.

File: src/test/java/com/hu/webdriver/tests/profile/ProfilePageTest.java [hunk 36]
```diff
@@ -0,0 +1,162 @@
...
+	public void beforeMethodNavigateToProfile(){
+		profilePage = loginHUAndNavigateToProfilePage();
+	}
+	/**
+	 * Test Case to verify Enter BIO.
+	 */
+	@Test(description = "Test Case to verify Enter BIO.")
+	public void testEnterBIO(){
+		String bio = propertyUtil.getProperty("hu_bio");
```
Comment (VenkiDeveloper) [inline src/test/java/com/hu/webdriver/tests/profile/ProfilePageTest.java:27]: Get bio value from properties file.

@Venu value will be changed.

**Comment by VenkiDeveloper** [comment]
@venuarelli Updated all my review comments, Update comments  and lemme know.

File: src/test/java/com/hu/webdriver/tests/profile/ProfilePageTest.java (hunk 34 above)
Comment (venuarelli) [inline src/test/java/com/hu/webdriver/tests/profile/ProfilePageTest.java:17]: changed

File: src/test/java/com/hu/webdriver/tests/BaseTest.java (hunk 33 above)
Comment (venuarelli) [inline src/test/java/com/hu/webdriver/tests/BaseTest.java:83]: Method doc provided

File: src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java (hunk 32 above)
Comment (venuarelli) [inline src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java:32]: Method doc provided.

File: src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java (hunk 31 above)
Comment (venuarelli) [inline src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java:26]: Method doc  updated.

File: src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java (hunk 30 above)
Comment (venuarelli) [inline src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java:19]: Return description added

File: src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java (hunk 29 above)
Comment (venuarelli) [inline src/test/java/com/hu/webdriver/tests/BaseLoginWebDriverTest.java:12]: Provided class document

File: src/main/java/com/hu/webdriver/pages/profile/UpDateProfilePage.java (hunk 28 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/UpDateProfilePage.java:56]: return doc is added.

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 27 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:335]: Hard code values are changed

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 26 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:314]: Description provided for method.

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 25 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:325]: Mistake will be changed.

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 23 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:308]: Hard code values are changed.

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 15 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:161]: @param doc added

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 16 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:177]: @param doc updated

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 17 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:229]: @param added

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 18 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:237]: @param added

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 19 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:244]: @param added

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 20 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:281]: @param added

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 21 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:290]: @param added

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 22 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:299]: @param added

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 14 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:127]: Changed

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 13 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:45]: Mistake changed

File: src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java (hunk 12 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/profile/ProfilePage.java:55]: changed to WebElement

File: src/main/java/com/hu/webdriver/pages/messages/MessagesPage.java (hunk 11 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/messages/MessagesPage.java:9]: Method doc is provided.

File: src/main/java/com/hu/webdriver/pages/messages/MessagesPage.java (hunk 10 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/messages/MessagesPage.java:7]: Author doc is added

File: src/main/java/com/hu/webdriver/pages/login/HULoginPage.java (hunk 9 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/login/HULoginPage.java:48]: changed to newsfeedPage.

File: src/main/java/com/hu/webdriver/pages/login/HULoginPage.java (hunk 8 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/login/HULoginPage.java:47]: Changed to NewsFeed

File: src/main/java/com/hu/webdriver/pages/home/HUHomePage.java (hunk 7 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/home/HUHomePage.java:12]: Author doc is added

File: src/main/java/com/hu/webdriver/pages/BasePage.java (hunk 6 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/BasePage.java:52]: comment provided

File: src/main/java/com/hu/webdriver/pages/BasePage.java (hunk 5 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/BasePage.java:49]: comments provided

File: src/main/java/com/hu/webdriver/pages/NewsFeedPage.java (hunk 4 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/NewsFeedPage.java:71]: comments provided.

File: src/main/java/com/hu/webdriver/pages/NewsFeedPage.java (hunk 3 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/NewsFeedPage.java:54]: changed to findVisibleElement Method.

File: src/main/java/com/hu/webdriver/pages/NewsFeedPage.java (hunk 2 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/NewsFeedPage.java:48]: Messages is added.

File: src/main/java/com/hu/webdriver/pages/BasePage.java (hunk 1 above)
Comment (venuarelli) [inline src/main/java/com/hu/webdriver/pages/BasePage.java:44]: documentation is provided for all methods.

**Pull request merged**